tests/Nox.Integration.Tests/DatabaseIntegrationTests/PostgresIntegrationTests.cs

```diff
@@ -2,17 +2,19 @@
 using Nox.Types;
 using System.Globalization;
 using System.Text.Json;
-
+using Microsoft.EntityFrameworkCore;
 using TestWebApp.Domain;
 
 using DayOfWeek = Nox.Types.DayOfWeek;
 using Guid = Nox.Types.Guid;
 
 namespace Nox.Integration.Tests.DatabaseIntegrationTests;
 
-public class PostgresIntegrationTests : PostgresTestBase
+[System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "xUnit1000:Test classes must be public", Justification = "Disabled")]//*/
+/*
+public /**/  class PostgresIntegrationTests : PostgresTestBase
 {
-    //[Fact]
+    [Fact]
```

Comment:
you need to comment or skip before integration, because ci pipeline is not able to run the tests for now

Reply:
I converted this class to private so that its tests will be ignored.  I think it is easier to manage it with this approach. What do you think ?

[System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "xUnit1000:Test classes must be public", Justification = "Disabled")]//*/
/*
public /**/  class  PostgresIntegrationTests : PostgresTestBase